site2/website-next/src/pages/index.js

```diff
@@ -149,7 +148,10 @@ export default function Home() {
       pulsingWaves.classList.add("show-waves");
     }, 50);
   });
-
+  // gets blog posts
+  const recentPosts = require("../../.docusaurus/docusaurus-plugin-content-blog/default/blog-post-list-prop-default.json");
```

Comment:
Have you tried to run `npm run build`? I'm not sure if this will make the building fail, please take a test, thanks

Reply:
@urfreespace  I have. It worked as expected on the build version.